kedro_mlflow/framework/hooks/node_hook.py

```diff
@@ -51,6 +90,9 @@ def before_node_run(
         mlflow.log_params(params_inputs)
 
 
+mlflow_node_hooks = MlflowNodeHook()
```

Comment:
why is this necessary? Is it for auto registration?

Reply:
It's for registring hooks in .kedro.yml or in setuptools entrypoints.